schema/deploy/warehouse/identifier-set-use/data.sql

```diff
@@ -0,0 +1,17 @@
+-- Deploy seattleflu/schema:warehouse/identifier-set-use/data to pg
+-- requires: warehouse/identifier-set-use
+
+begin;
+
+insert into warehouse.identifier_set_use (use, description)
+    values
+        ('sample', 'Sample ID'),
+        ('collection', 'Collection ID'),
+        ('clia', 'CLIA ID'),
+        ('kit','Test kit ID'),
+        ('test-strip','Test strip ID')
```

Comment:
I'd suggest updating these descriptions to be a bit more useful, a la the table/column comments in our schema or the existing identifier set descriptions. I think future folks will appreciate it.

Reply:
Yes, I think you're right. Updated them to be more "descriptive".

Reply:
Thanks!